source/Octopus.Tentacle.Tests.Integration/Support/HalibutTimeoutsAndLimitsForTestBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using Halibut.Diagnostics;
+
+namespace Octopus.Tentacle.Tests.Integration.Support
+{
+    public class HalibutTimeoutsAndLimitsForTestBuilder
+    {
+
+        public HalibutTimeoutsAndLimits Build()
+        {
+            var halibutTimeoutAndLimits = HalibutTimeoutsAndLimits.RecommendedValues();
+            halibutTimeoutAndLimits.TcpKeepAliveEnabled = true;
```

Comment:
`TcpKeepAliveEnabled` are on by default in `HalibutTimeoutsAndLimits.RecommendedValues()` as `HalibutTimeoutsAndLimits` defaults it to true

Reply:
Ah I had not realised that was the case.

Reply:
Save on an extra line of code 😆